.github/workflows/ci.yml

```diff
@@ -6,6 +6,32 @@ on:  # yamllint disable-line rule:truthy
   pull_request:
 
 jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Fetch the src
+      uses: actions/checkout@v3
+    - name: Set up Python 3.11
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.11
+
+    - name: Install requirements
+      run: >-
+        python -m pip install
+        -r requirements/base.in
+        -c requirements/base.txt
```

Comment:
These shouldn't be separate, otherwise, they'll be overwritten by the following `pip install` with possibly different versions of things.

Reply:
> These shouldn't be separate, otherwise, they'll be overwritten by the following `pip install` with possibly different versions of things.

I'm sorry, but I don't clearly understand how the installation of requirements should look in the configuration file. Here is your previous fix https://github.com/kpi-web-guild/django-girls-blog-OlenaEfymenko/pull/31#discussion_r1425681784. 
I'll try to find some examples and docs, maybe.

Reply:
`run: python -m pip install -r requirements/base.in -c requirements/base.txt`
like this variant

Reply:
You mustn't install base explicitly/directly.

Reply:
Yes, I understood. Thank you)

Reply:
> Here is your previous fix [#31 (comment)](https://github.com/kpi-web-guild/django-girls-blog-OlenaEfymenko/pull/31#discussion_r1425681784).

That was just an example of a formatting change, orthogonal to the fact that the step shouldn't be included, actually.

Reply:
Marking this as resolved.